src/sentry/issues/producer.py

```diff
@@ -58,5 +67,32 @@ def produce_occurrence_to_kafka(
             payload_data["event"] = event_data
         payload = KafkaPayload(None, json.dumps(payload_data).encode("utf-8"), [])
         _occurrence_producer.produce(Topic(settings.KAFKA_INGEST_OCCURRENCES), payload)
+    elif payload_type == PayloadType.STATUS_CHANGE:
+        if not status_change:
+            raise ValueError("status_change must be provided")
+
+        organization = Project.objects.get(id=status_change.project_id).organization
+        if not features.has("organizations:issue-platform-api-crons-sd", organization):
+            return
+
+        grouphash = (
+            GroupHash.objects.filter(
+                project=status_change.project_id,
+                hash__in=status_change.fingerprint,
```

Comment:
should we just pull the first `fingerprint` from the array? 

see https://github.com/getsentry/sentry/blob/3c6bdaafdf16f5945c84fd46acce628f1b052fd6/src/sentry/issues/ingest.py#L163